modules/drivers/databricks/test/metabase/driver/databricks_test.clj

```diff
@@ -45,102 +45,112 @@
-                :table-name "orders"
-                :pk? false
-                :name "user_id"
-                :database-type "int"
-                :database-position 1
-                :base-type :type/Integer
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "product_id"
-                :database-type "int"
-                :database-position 2
-                :base-type :type/Integer
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "subtotal"
-                :database-type "double"
-                :database-position 3
-                :base-type :type/Float
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "tax"
-                :database-type "double"
-                :database-position 4
-                :base-type :type/Float
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "total"
-                :database-type "double"
-                :database-position 5
-                :base-type :type/Float
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "discount"
-                :database-type "double"
-                :database-position 6
-                :base-type :type/Float
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "created_at"
-                :database-type "timestamp"
-                :database-position 7
-                :base-type :type/DateTimeWithLocalTZ
-                :json-unfolding false}
-               {:table-schema "test-data"
-                :table-name "orders"
-                :pk? false
-                :name "quantity"
-                :database-type "int"
-                :database-position 8
-                :base-type :type/Integer
-                :json-unfolding false}}
-             (reduce conj #{} (driver/describe-fields :databricks (mt/db) {:schema-names ["test-data"]
-                                                                           :table-names ["orders"]})))))))
+  (mt/test-driver
+    :databricks
+    (let [fields (vec (driver/describe-fields :databricks (mt/db) {:schema-names ["test-data"]
+                                                                   :table-names ["orders"]}))]
+      ;; Currently we have `test-data` in `metabase_ci` and `metabase_drivers`. Ensure the underlying sql query
+      ;; does not waste resources by returning for both.
```

Comment:
Is it really just about wasting? I thought it would be a bug to mix fields from different catalogs...